.cirrus.yml

```diff
@@ -27,6 +27,18 @@ task:
     - cd Tests/Packet++Test && Bin/Packet++Test
     - cd ../Pcap++Test && Bin/Pcap++Test -n
 
+task:
+  name: Regression Tests Linux
+  container:
+    image: zlowram/ubuntu1910-clang:latest
```

Comment:
can you check if the regression can run on the `seladb/ubuntu1804` container?

I can add `clang` to this image if needed

Reply:
It shouldn't be a problem to run there if there's `clang`. 

Please, make sure to install the latest version of `clang` available in the repositories (it should be `clang-9`) and that the `clang` command is available. Installing `clang-9` on Ubuntu 18.04 does not create `clang`, but `clang-9`. A symbolic link should fix that.

Reply:
Let me know when you update the image and I'll change it. Or feel free to merge it and change it yourself if you want.

Reply:
does it have to be `clang-9`? will it work if I install the default `clang` version that is installed with `apt-get install clang`?

Reply:
@zlowram before I make changes to the image please let me know if `clang-9` is specifically required or any `clang` version will do

Reply:
Oh sorry, I missed this one. In OSSFuzz they use the Clang version closest to trunk. Right now it's Clang10 so my recommendation would be to use the latest version available in the distro repositories.

Reply:
sure, I'll install `clang-9`. But please note the executable name will probably be `clang-9` instead of `clang`. Is this ok?

Reply:
The value I'm giving to `CC`  and `CXX` for the fuzzing build when they're not overriden by the user is `clang` and `clang++`, so I'd would be good to have them.

A symbolic link for both `clang` and `clang++` would do the trick for this.

Reply:
this is done. You can use the image `seladb/ubuntu1804`. It has `clang-9` and I added the following symbolic links:

- `clang-9` --> `clang`
- `clang++-9` --> `clang++`

Reply:
Awesome, I already changed it and [works like a charm](https://cirrus-ci.com/task/6696600401608704).